pkg/asset/installconfig/gcp/validation.go

```diff
@@ -79,7 +79,15 @@ func validateInstanceAndDiskType(fldPath *field.Path, diskType, instanceType, ar
 		return nil
 	}
 
+	if strings.Contains(instanceType, "custom") {
+		logrus.Debugf("custom instance type %s was selected", instanceType)
+	}
+
```

Comment:
Do we still need this check? It doesn't really matter if the instance type is `custom` as long as we can determine the VM family it belongs to.